backend/dataall/modules/datasets_base/services/dataset_service_interface.py

```diff
@@ -50,3 +50,7 @@ def extend_attach_steward_permissions(session, dataset, new_stewards) -> bool:
     def extend_delete_steward_permissions(session, dataset, new_stewards) -> bool:
         """Abstract method to be implemented by dependent modules that want to attach additional permissions to Dataset stewards"""
         ...
+
```

Comment:
What is this method used for?

Reply:
I switched the name to `find_consumption_details_resource_uri` and forgot to change here but only changed on `S3DatasetService` - good catch

It will likely go away anyway - was used for Preview on Confidential data

Reply:
removed as part of removing Preview Tab for Confidential Data